packages/waku/src/cli.ts

```diff
@@ -58,7 +58,7 @@ if (values.version) {
     case 'build':
       runBuild({
         ssr: withSsr,
-        vercel: values['with-vercel'],
+        vercel: values['with-vercel'] ?? !!process.env.VERCEL,
```

Comment:
We already have it in `.../builder/build.ts`.
Should we move it here? Yeah, that makes sense probably.